src/style.css

```diff
@@ -1,3 +1,68 @@
 body {
   margin: 0;
 }
+
+/* navigation */
+
+.header {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  min-width: 1024px;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.logo {
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+
+  list-style: none;
+}
+
+.nav__list > li:first-child a {
+  color: #00acdc;
```

Comment:
you're using this color 3 times so create css variable for it 
[checklist](https://github.com/mate-academy/layout_moyo-header?tab=readme-ov-file#checklist): CSS Variable is used for a blue color